src/backend/wine/manager/downloader/__tests__/utilities/rest.test.ts

```diff
@@ -29,7 +30,7 @@ describe('Utilities - Rest', () => {
     expect(() => {
       unlinkFile(__dirname)
     }).toThrowError(
-      `Couldn't remove ${workDir}/src/backend/wine/manager/downloader/__tests__/utilities!`
+      `Couldn't remove ${workDir}${path.sep}src${path.sep}backend${path.sep}wine${path.sep}manager${path.sep}downloader${path.sep}__tests__${path.sep}utilities!`
```

Comment:
we should use `path.join` here like `path.join(workDir, 'src', 'backend', ...)` instead interpolating `path.sep`, it's easier to read and should take care of using the right separator

Reply:
Thanks, that is much more readable!